src/pages/ProfilesExplorerView/components/SceneProfilesExplorer/components/Header.tsx

```diff
@@ -45,6 +60,10 @@ export function Header(props: HeaderProps) {
         </div>
 
         <div className={styles.appControlsRight}>
+          {InsightsLauncher && (
+            <InsightsLauncher dataSourceUid={dataSourceVariable.getValueText()} serviceName={serviceName} />
+          )}
```

Comment:
Let's wrap it in ErrorBoundry to avoid breaking the app in case InsightLauncher throws an error

```suggestion
          {InsightsLauncher && (
            <ErrorBoundary>
              {({ error }) =>
                error ? undefined : (
                  <InsightsLauncher dataSourceUid={dataSourceVariable.getValueText()} serviceName={serviceName} />
                )
              }
            </ErrorBoundary>
          )}
```